cipher/ecc.c

```diff
@@ -1613,10 +1650,35 @@ ecc_decrypt_raw (gcry_sexp_t *r_plain, gcry_sexp_t s_data, gcry_sexp_t keyparms)
       goto leave;
     }
 
-
   ec = _gcry_mpi_ec_p_internal_new (sk.E.model, sk.E.dialect, flags,
                                     sk.E.p, sk.E.a, sk.E.b);
 
+  /* For GOST extract the UKM value. */
+  if ((flags & PUBKEY_FLAG_GOST) || 0 == strncmp ("GOST", sk.E.name, 4))
+    {
+      // FIXME: Expect an uncompressed point format 0x04...
```

Comment:
C++-style comment.
Also would you like to fix the FIXME?

Reply:
* OK, I'll use `/* */` instead.
* Probably no, it's just a note to ease debugging due to a possible change of the input point format in the future.

Reply:
Then it should not be a FIXME

Reply:
Dropped `FIXME`.